packages/stark-build/config/webpack-partial.common.js

```diff
@@ -80,7 +89,9 @@ module.exports = metadata => {
 						{
 							loader: "tslint-loader",
 							options: {
-								typeCheck: false // FIXME remove this line once the type checking issues are gone (cfr FIXME above)
+								typeCheck: false, // FIXME remove this line once the type checking issues are gone (cfr FIXME above)
```

Comment:
Can you replace "remove this line" by "enable this option"? Because the option is by default `false`